common/resource_manager.cc

```diff
@@ -583,7 +583,7 @@ std::string ResourceManager::DecryptResource(const std::string& path) {
   rewind(src);
 
   // Read buffer from the source file
-  std::unique_ptr<char> src_buf(new char[src_len]);
+  std::unique_ptr<char[]> src_buf(new char[src_len]);
```

Comment:
It already changed by

commit 218a64dae6d8e57bbda7cea732fd972c26a1764d
Author: sung-su.kim <sung-su.kim@samsung.com>
Date:   Thu Oct 22 09:38:27 2015 +0900

    Fix incorrectly unique_ptr declared
    
    from unique_ptr<char> to unique_ptr<char[]>